apache-rat-core/pom.xml

```diff
@@ -15,113 +15,134 @@
-        <directory>src/main/resources</directory>
-      </resource>
-      <resource>
-        <filtering>true</filtering>
-        <directory>src/main/filtered-resources</directory>
-      </resource>
-      <resource>
-        <directory>..</directory>
-        <targetPath>META-INF</targetPath>
-        <includes>
-          <include>RELEASE_NOTES.txt</include>
-        </includes>
-      </resource>
-    </resources>
+	<build>
+		<resources>
+			<resource>
+				<filtering>false</filtering>
+				<directory>src/main/resources</directory>
+			</resource>
+			<resource>
+				<filtering>true</filtering>
+				<directory>src/main/filtered-resources</directory>
+			</resource>
+			<resource>
+				<directory>..</directory>
+				<targetPath>META-INF</targetPath>
+				<includes>
+					<include>RELEASE_NOTES.txt</include>
+				</includes>
+			</resource>
+		</resources>
 
-    <pluginManagement>
-      <plugins>
-        <plugin>
-          <groupId>org.apache.rat</groupId>
-          <artifactId>apache-rat-plugin</artifactId>
-          <configuration>
-            <excludes>
-              <!-- These files have bad license headers because they are used to test bad license headers -->
-              <exclude>src/test/resources/appliedAL20/bad/*.*</exclude>
-              <exclude>src/test/resources/oasis/bad/*.*</exclude>
-              <!-- These files does not have license headers because they are used to test license headers -->
-              <exclude>src/test/resources/elements/Source.java</exclude>
-              <exclude>src/test/resources/elements/ILoggerFactory.java</exclude>
-              <exclude>src/test/resources/elements/sub/Empty.txt</exclude>
-              <exclude>src/test/resources/violations/bad.txt</exclude>
-              <exclude>src/test/resources/violations/FilterTest.cs</exclude>
-              <!-- used to test for binary files; does not work on UTF-8 (e.g. MacOSX) -->
-              <exclude>src/test/resources/binaries/Image-png.not</exclude>
-              <!-- These files have a valid license header that was not recognised by older rat versions -->
-              <exclude>src/test/resources/elements/TextHttps.txt</exclude>
-              <!-- Remove after using RAT 0.14 for dog-fooding ..... older versions do not ignore JSON -->
-              <exclude>src/test/resources/elements/plain.json</exclude>
-            </excludes>
-          </configuration>
-        </plugin>
-      </plugins>
-    </pluginManagement>
-  </build>
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<groupId>org.apache.rat</groupId>
+					<artifactId>apache-rat-plugin</artifactId>
+					<configuration>
+						<excludes>
+							<!-- These files have bad license headers because
+							they are used to test bad license headers -->
+							<exclude>src/test/resources/appliedAL20/bad/*.*</exclude>
+							<exclude>src/test/resources/oasis/bad/*.*</exclude>
+							<!-- These files does not have license headers
```

Comment:
Typo: These files do not ....
(must have been in there for a couple of years ;))